ddl/delete_range.go

```diff
@@ -238,6 +238,20 @@ func insertJobIntoDeleteRangeTable(ctx sessionctx.Context, job *model.Job) error
 		}
 	case model.ActionDropTable, model.ActionTruncateTable:
 		tableID := job.TableID
+		var partitionIDs []int64
+		if err := job.DecodeArgs(&partitionIDs); err != nil {
+			return errors.Trace(err)
```

Comment:
Maybe we can try to decode it as []byte again here, to adapt to the old TiDB binary? @zimulala

Reply:
@tiancaiamao 
I think we can use the method of appending `partitionIDs` to Args directly.

Reply:
@tiancaiamao @zimulala Done.